concent_api/core/tests/test_integration_get_task_result.py

```diff
@@ -999,3 +999,116 @@ def test_concent_requests_task_result_from_provider_and_requestor_receives_task_
         self.assertEqual(message_from_concent.file_transfer_token.operation, FileTransferToken.Operation.download)
 
         self._assert_client_count_is_equal(2)
+
+    def test_concent_requests_task_result_from_provider_when_compute_task_def_deadline_is_float(self):
```

Comment:
This tests is pretty much, one big Code Duplication of `test_concent_requests_task_result_from_provider_and_requestor_receives_force_get_task_upload_because_file_already_uploaded`

Reply:
Test deleted